common/README.md

```diff
@@ -0,0 +1,79 @@
+# Protos for Ruby API Client Generator
+
+This directory is the repository for the shared protos and their binary files,
+as well as the mechanism to generate new binary files.
+
+## Usage
+### Install the Proto Compiler
+This generator relies on the Protocol Buffer Compiler to [orchestrate] the
+client generation.
+
+```sh
+$ git clone https://github.com/googleapis/gapic-generator-ruby.git
+$ cd gapic-generator-ruby
+$ git submodule update --init
+
+$ cd proto
+
+
+# Declare the protobuf version to use.
+$ export PROTOBUF_VERSION=3.6.1
+
+# Declare the target installation system.
+# export SYSTEM=osx
+$ export SYSTEM=linux
+
+# Get the precompiled protobuf compiler.
+$ curl --location https://github.com/google/protobuf/releases/download/v${PROTOBUF_VERSION}/protoc-${PROTOBUF_VERSION}-linux-x86_64.zip > usr/src/protoc/protoc-${PROTOBUF_VERSION}.zip
+$ cd /usr/src/protoc/
+$ unzip protoc-${PROTOBUF_VERSION}.zip
+$ rm protoc-${PROTOBUF_VERSION}.zip
+
+# Link the protoc to the path.
+$ ln -s /usr/src/protoc/bin/protoc /usr/local/bin/protoc
+$ mkdir -p /protos/
+
+# Move the common protobuf files to the local include folder.
+$ cp -R /usr/src/protoc/include/* /usr/local/include/
```

Comment:
I know this isn't new, but since we're reorganizing, could we add the `grpc-tools`gem as a development dependency or something similar that would make this setup easier?

Reply:
This is a copy of the gapic-generator README. I don't know anything about this. @landrito would know more than me.

I can revise the README to be more specific to how the directory is being used.

Reply:
Let's deal with it later then. I don't really like asking people to do things in `/usr/src/`, etc. but it's a documentation thing we can address at some point.

Reply:
TBH I didn't like it either, and had the same concerns. But this file here is a copy/paste. I'll update the README and take this out.